### PR TITLE
Parse only selected sections in NormalizeSeparators

### LINKED ISSUE #116: option for NormalizeSeparators to discard particular sections
let's say I have Settings and Variables sections already aligned by other transformers, then running NormalizeSeparators transformer breaks that previous alignment.
It affects me especially when having pre-commit hook with seperate hooks per transformer, there is an infinite loop of refactors done by these transformers and there's no way to commit without '--no-verify' option.

### PR DESCRIPTION
**Pull request by bhirsz**
Closes #116

### CODE REVIEW

File: robotidy/transformers/NormalizeSeparators.py [hunk 1]
```diff
@@ -15,15 +16,68 @@ class NormalizeSeparators(ModelTransformer):
...
+            'variables'
+        }
+        if sections is None:
+            return default
+        if not sections:
+            return {}
+        splitted = sections.split(',')
+        parsed_sections = set()
+        for split in splitted:
+            # normalization to also accept test_cases, setting, variable
```
Comment (mnojek) [inline robotidy/transformers/NormalizeSeparators.py:44]: Is this comment needed?

File: robotidy/transformers/NormalizeSeparators.py [hunk 2]
```diff
@@ -15,15 +16,68 @@ class NormalizeSeparators(ModelTransformer):
...
+        return node
+
+    def visit_CommentSection(self, node):  # noqa
+        return self.should_visit('comments', node)
+
+    def visit_SettingSection(self, node):  # noqa
+        return self.should_visit('settings', node)
+
+    def visit_VariableSection(self, node):  # noqa
+        return self.should_visit('settings', node)
```
Comment (mnojek) [inline robotidy/transformers/NormalizeSeparators.py:73]: 'variables'?

Comment (bhirsz) [reply robotidy/transformers/NormalizeSeparators.py:73]: "it is fine to only test edge cases" ugh :D Thanks!

**Review by mnojek — APPROVED** [review]
